examples/worlds/multi_lrauv_race.sdf

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 
 <!-- Run demo with ign-gazebo examples/standalone/multi_lrauv_race
-  (example is provided in ign-gazebo6+). -->
+  (example is provided in ign-gazebo7+). -->
```

Comment:
I think this can be reverted.

Reply:
reverted. c0657e52630f5020644d814ca7f2a391d44d9f49